browser/ui/color/brave_color_mixer.cc

```diff
@@ -702,8 +702,6 @@ void AddBraveOmniboxColorMixer(ui::ColorProvider* provider,
 
   // We don't use bg color for location icon view.
   mixer[kColorPageInfoBackground] = {SK_ColorTRANSPARENT};
-
-  AddOmniboxHoverSelect(mixer);
```

Comment:
We now take the Chromium colors which is much simpler - we still need this for Private/Tor